src/App.js

```diff
@@ -1,14 +1,127 @@
 import React, { Component } from 'react';
-import './App.css';
+import axios from 'axios';
+import MovieList from './MovieList';
+import SearchBar from './SearchBar';
+// import MovieShowcase from './MovieShowcase';
+
+
+// Movie List app components:
+// search, movie poster, selected movie screen,
+// favorites, home page
+// api key 63cc1471f10ab940a7de31d3ef1b14d9
+// api read access token
+// eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJhdWQiOiI2M2NjMTQ3MWYxMGFiOTQwYTdkZTMxZDNlZjFiMTRkOSIsInN1YiI6IjU4OWQxNGNiYzNhMzY4NWY4YjAwMGFjZCIsInNjb3BlcyI6WyJhcGlfcmVhZCJdLCJ2ZXJzaW9uIjoxfQ.ce1Bttr4z7FpNXCrctfTFoy7GaSBrjg5lJWx-K6XpNU
+// example api request https://api.themoviedb.org/3/movie/550?api_key=63cc1471f10ab940a7de31d3ef1b14d9
+
 
 class App extends Component {
+  constructor() {
+    super();
+
+    this.state = {
+      searchText: '',
+      returnedMovies: [],
+      favorites: []
+    };
+  }
+
+
+
+  componentDidMount() {
+    axios.get('http://localhost:4000/movies')
+        .then(resp => {
+          this.setState({
+            favorites: resp.data,
+
+
+          });
+        })
+        .catch(err => {
+          return ('No favorites yet!');
+        });
+
+  }
+
+  // Adding a Favorite!
+  handleAddFavorite(attributes) {
+    axios.post('http://localhost:4000/movies', attributes)
+    .then(resp => {
+      this.setState({
+        favorites: [...this.state.favorites, resp.data]
+      });
+    })
+    .catch(err => console.log(err));
+  }
+
+
+  // Deleting a Favorite. We all make mistakes.
+  handleDeleteFavorite() {
+    axios.delete('http://localhost:4000/favorites')
+    .then(() => {
+      this.setState();
```

Comment:
Also, `this.setState()` doesn't do anything since we aren't passing anything to update...